tests/server/blueprints/cases/test_cases_views.py

```diff
@@ -155,7 +155,7 @@ def test_causatives(app, user_obj, institute_obj, case_obj):
             institute=institute_obj,
             case=case_obj,
             user=user_obj,
-            link='causative_var_link',
+            link='causative_var_link/{}'.format(variant_obj['_id']),
```

Comment:
Yep!

Reply:
Thanks! After the tests pass I'm going to merge!

Reply:
🤞

Reply:
YESSSS